### ISSUE TITLE
WCAG Guidelines

### ISSUE THREAD

**Issue by acrose99** [acrose99/PersonalWebsite#4]
# Web Accessibility is very important and something I need to begin to implementing.

Honestly, it never even crossed my mind (even as someone with a disability). It should be mentioned more.

I've done an automatic check and will begin to investigate firther

[Check these](https://github.com/acrose99/PersonalWebsite/files/5269710/achecker_2020-09-23_16-51-54.pdf)
